docusaurus/docs/running-tests.md

```diff
@@ -180,21 +180,21 @@ As an alternative or companion to `enzyme`, you may consider using `react-testin
 To install `react-testing-library` and `jest-dom`, you can run:
 
 ```sh
-npm install --save react-testing-library jest-dom
+npm install --save-dev @testing-library/react jest-dom
```

Comment:
This is supposed to be `--save`, not `--save-dev`. We intentionally install everything as regular dependencies.